src/libraries/System.Security.Cryptography.Xml/tests/TestHelpers.cs

```diff
@@ -185,12 +185,34 @@ public static IEnumerable<SymmetricAlgorithmFactory> GetSymmetricAlgorithms(bool
-
         public static X509Certificate2 GetSampleX509Certificate()
         {
-            return new X509Certificate2(SamplePfx, "mono");
+            byte[] key = Convert.FromBase64String(@"
+MIICWQIBAAKBgQCklg/n5bi6DYjx5r1sU0zDtXjC23e/aMQIqSZZU2S9x1Guf146
+AYwaDnSNUR1dUOwIVqbtf/Hud10i/SObG9enp/WNX7dT72XIqY9SHLhPv2BtK3rq
+Cv27RyYnjLjuhvh9RQnrAR1A74TOWaOUFtPPbgkPVUPMfZOchcwc01kiSQIBEQKB
+gEPFUdf1LfJBzvox1Yb1H5vhT9fDx+VnX8dUtXATR5lwMLFDgSbxlAq6qHZssb0D
+NANu+XDaVI9eU4bgtE7tWMvlzCY3rczilxbBMVIxrQJtSDjQehHDsh5dOHEkZ7S6
+6sGcTogQmihpp1nWF/MIWWBlSWWPaSNQoTRSZs1ULsNxAkEAsss5H579+QO1SBFA
+/lC3dDZFJ4GSSJT+QiOelhyoyOqC8x8SaD/i2s0+K3gKzHz73MywxKBsT5JRoDMk
+McW30QJBAOuoQJRNrYbzJUMq28pVtvifVOUaCAu4KiKeCHmgakFBkdZQsrmwIIHX
+/yY6hDYQDgj31aIRwUWeXh16vNUh2PkCQBUI2Ys/4aTTQoDy+JZj2Vj3U24PPmLk
+WiXmEqg/m2LuacI/5AxDwFX6B1Boeb20Wd29unF8SPpNcwPJyAXbBpECQEVPmoX4
+upEaVkDufOEoRN+2ZENiAl3MwRk9igWnpsfmG9WfQ6AGoCYweDhrkEwi1vOUL8ZB
+dRR533IVClzNuEkCQFnaqt/8FRThg6hBAFoW7JCmQQeB8LTU9M4PKkWZ6UspG9v/
+9SG9cNLdMjx8vOS4K5/eWdwN/22vbvCexK98kiE=");
+
+            using (RSA rsa = RSA.Create())
+            {
+                rsa.ImportRSAPrivateKey(key, out _);
+
+                var certReq = new CertificateRequest("CN = Poupou's-Software-Factory", rsa, HashAlgorithmName.SHA256, RSASignaturePadding.Pkcs1);
+
+                var now = DateTimeOffset.Now;
+                var oneHourLater = now;
+                oneHourLater.AddHours(1);
```

Comment:
AFAIK `AddHour` returns the update value but doesn't change itself ==> `oneHourLater` will stay as `now`.